### ISSUE TITLE
PHPCBF adds brace for ELSE IF split over multiple lines

### ISSUE THREAD

**Issue by meijerm15** [squizlabs/PHP_CodeSniffer#558]
PHPCBF is currently fixing bracketless if-else statements incorrectly and causing a fatal error.
Which means that the code won't run, because of the 2 else statements in a row, as seen in my example.

**Before**:
```php
     public function test($test)
     {
        if ($test) {

        } else
            if ($test) {

            } else {

            }
     }
```
**After**:
```php
     public function test($test)
{
    if ($test) {
    } else {
        if ($test) {
        }
    } else {
    }
        }
```
**Log:**
Processing test.php [PHP => 53 tokens in 12 lines]... DONE in 3ms (9 fixable violations)
        => Fixing file: 0/9 violations remaining [made 6 passes]... DONE in 61ms
Patched 1 file
Time: 146ms; Memory: 5.25Mb

**Version:**
PHP_CodeSniffer version 2.3.0 (stable) by Squiz (http://www.squiz.net)

**Comment by aik099** [comment]
Can you please specify standard used and error reported in original code.

**Comment by meijerm15** [comment]
@aik099 

**Command**
phpcbf --standard=psr2 -n test.php 

So I've used the psr2 standard and could you explain what you meant by 'error reported in original code.' because this test snippet does reproduce the fatal code change

**Comment by aik099** [comment]
> could you explain what you meant by 'error reported in original code.' because this test snippet does reproduce the fatal code change

This is result of `phpcs -s --standard=psr2 -n test.php` command. The `phpcbf` is only fixing what `phpcs` is reporting. The `-s` option would show sniff code, that is responsible for the fix.

**Comment by meijerm15** [comment]
**Snippet**
```php
<?php
namespace test;

class Test
{
     public function test($test)
     {
        if ($test) {

        } else
            if ($test) {

            } else {

            }
     }
}
```
**Command**
 phpcs -s --standard=psr2 -n test.php 

 8 | ERROR | [x] Expected 1 newline after opening brace; 2 found
    |       |     (Squiz.ControlStructures.ControlSignature.NewlineAfterOpenBrace)
 10 | ERROR | [x] Expected 1 space after ELSE keyword; newline found
    |       |     (Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword)
 10 | ERROR | [x] Inline control structures are not allowed
    |       |     (Generic.ControlStructures.InlineControlStructure.NotAllowed)
 11 | ERROR | [x] Line indented incorrectly; expected 8 spaces, found
    |       |     12 (Generic.WhiteSpace.ScopeIndent.IncorrectExact)
 11 | ERROR | [x] Expected 1 newline after opening brace; 2 found
    |       |     (Squiz.ControlStructures.ControlSignature.NewlineAfterOpenBrace)
 13 | ERROR | [x] Expected 1 newline after opening brace; 2 found
    |       |     (Squiz.ControlStructures.ControlSignature.NewlineAfterOpenBrace)
 16 | ERROR | [x] Line indented incorrectly; expected 8 spaces, found
    |       |     5 (Generic.WhiteSpace.ScopeIndent.IncorrectExact)

**Comment by gsherwood** [comment]
This was a problem with the inline control structure sniff not understanding that the `else if` split over multiple lines was actually the same statement. This is now fixed.

You need to run PHPCBF a couple of times over this specific code block because the sniff to get it perfect (due to a conflict between the indent, newline and elseif sniffs in this specific case) but you will end up with this:

```php
<?php
namespace test;

class Test
{
     public function test($test)
     {
        if ($test) {
        } elseif ($test) {
        } else {
        }
     }
}
```
